src/mod.ts

```diff
@@ -22,12 +23,86 @@ export const run = async function (functionDir: string) {
     return potentialFunctionFiles;
   });
 
-  // The CLI expects a JSON payload to be output to stdout
-  // This is formalized in the `run` hook of the CLI/SDK Tech Spec:
-  // https://corp.quip.com/0gDvAsqoaaYE/Proposal-CLI-SDK-Interface#temp:C:fOC1991c5aec8994d0db01d26260
-  console.log(JSON.stringify(resp || {}));
+  return resp || {};
+};
+
+// Start a http server that listens on the provided port
+// This server exposes two routes
+//  GET  `/health`    Returns a 200 OK
+//  POST `/functions` Accepts event payload and invokes `run`
+const startServer = async function (port: number) {
+  let server;
+  try {
+    server = Deno.listen({ port });
+  } catch (e) {
+    throw new Error(`Unable to listen on ${port}, got ${e}`);
+  }
+  for await (const conn of server) {
```

Comment:
Async iterables are wild to me - that it will run the loop for every new connection that comes in.

Reply:
yeah it's strange to me too, at first I used 
`import { serve } from "https://deno.land/std@0.149.0/http/server.ts";`
But then noticed that API is deprecated and this is the recommended way now.